doc/api/errors.md

```diff
@@ -450,18 +450,35 @@ added properties.
 
 #### error.code
 
-Returns a string representing the error code, which is always `E` followed by
+Is a string representing the error code, which is always `E` followed by
 a sequence of capital letters, and may be referenced in `man 2 intro`.
 
 #### error.errno
 
-Returns a number corresponding to the **negated** error code, which may be
+Is a number corresponding to the **negated** error code, which may be
 referenced in `man 2 intro`. For example, an `ENOENT` error has an `errno` of
 `-2` because the error code for `ENOENT` is `2`.
 
+In some modules, this property is the same string as `error.code`.
```

Comment:
OK, then the docs should say "In some modules, is a number corresponding to ...etc." because the first sentence is being contradicted by the paragraph after ATM.

Is there an open issue to track this? It doesn't strike me as so helpful to sometimes be an "E" string, and sometimes a number.